litellm/proxy/auth/auth_checks.py

```diff
@@ -263,15 +268,57 @@ async def get_user_object(
             return cached_user_obj
     # else, check db
     try:
-
         response = await prisma_client.db.litellm_usertable.find_unique(
             where={"user_id": user_id}
         )
 
         if response is None:
             if user_id_upsert:
+                # grab the default if exists
+                user_to_add = {"user_id": user_id}
+
+                # check if the user_email_jwt_field is set
+                if user_email:
+                    user_to_add["user_email"] = user_email
+
+                # check if a default team is set, if it is, associated it - create it otherwise
+                if team_name_default:
+                    try:
+                        # if the default team exists, verify it exists, if not upsert it
+                        response = await prisma_client.db.litellm_teamtable.find_unique(
+                            where={"team_alias": team_name_default}
+                        )
+
+                        print("RESPONSE AFTER FINDING TEAM FROM ALIAS")
```

Comment:
can we cleanup the print statements - it'll fail our linting

Reply:
Yes I'll clean this up. Done